index.html

```diff
@@ -48,30 +51,182 @@ <h2>I'm a Software Developer</h2>
+           <p class="flex-item item3 proitem3">bootstrap</p>
+          </div>
+        </div>
+        <button class="work-button probutton">See Project</button>
+        <div id="project2">
+          <h2 class="work-subtitle protitle pro2title">Cafe Bill</h2>
+          <p class="work-details prodetails">Lorem ipsum dolor sit amet consectetur adipisicing elit. Vero aliquid debitis nulla? Nulla, doloribus id consequatur.</p>
+          <div class="flex pro2flex">
+           <p class="flex-item item1 proitem1">css</p>
+           <p class="flex-item item2 proitem2">html</p>
+           <p class="flex-item item3 proitem3">bootstrap</p>
+          </div>
+        </div>
+        <button class="work-button probutton">See Project</button>
+      </section>
+      <section class="projsecondpage">
+        <div id="project3">
+          <h2 class="work-subtitle protitle">Cat's Photo</h2>
+          <p class="work-details prodetails">Cat's Photo Project built on freecodecamp.org</p>
+          <div class="flex">
+           <p class="flex-item item1 proitem1">css</p>
+           <p class="flex-item item2 proitem2">html</p>
+           <p class="flex-item item3 proitem3">bootstrap</p>
+          </div>
+        </div>
+        <button class="work-button probutton">See Project</button>
+        <div id="project4">
+          <h2 class="work-subtitle protitle pro2title">Photo Gallery</h2>
+          <p class="work-details prodetails">Cat's Photo Gallery implementation of CSS flexbox</p>
+          <div class="flex pro2flex">
+           <p class="flex-item item1 proitem1">css</p>
+           <p class="flex-item item2 proitem2">html</p>
+           <p class="flex-item item3 proitem3">bootstrap</p>
+          </div>
+        </div>
+        <button class="work-button probutton">See Project</button>
+      </section>
+      <section class="projthirdpage">
+        <div id="project5">
+          <h2 class="work-subtitle protitle">The Quiz</h2>
+          <p class="work-details prodetails">Make a quiz templete using html forms</p>
+          <div class="flex">
+           <p class="flex-item item1 proitem1">css</p>
+           <p class="flex-item item2 proitem2">html</p>
+           <p class="flex-item item3 proitem3">bootstrap</p>
+          </div>
+        </div>
+        <button class="work-button probutton">See Project</button>
+        <div id="project6">
+          <h2 class="work-subtitle protitle pro2title">Tribute Page</h2>
+          <p class="work-details prodetails">Tribute to Dr. Abdul Salam a Nobel Scientist.</p>
+          <div class="flex pro2flex">
+           <p class="flex-item item1 proitem1">css</p>
+           <p class="flex-item item2 proitem2">html</p>
+           <p class="flex-item item3 proitem3">bootstrap</p>
+          </div>
+        </div>
+        <button class="work-button probutton">See Project</button>
+      </section>
+      <section id="aboutme">
+        <h1 id="me">About me</h1>
```

Comment:
- [ ]  H1 carries a title-like message for other devices such as screen readers, so it is professionally advisable to use h1 once on a webpage and you will agree with me that h1, which carries a title-like message should not be used for about me heading. I would advise you to use another hx for the satisfaction of this reason.